lib/rubocop/cop/commissioner.rb

```diff
@@ -25,6 +25,11 @@ def correctors
           @correctors ||= cop_reports.map(&:corrector)
         end
 
+        # @return [Corrector]
+        def merged_correctors
+          correctors.compact.inject(:merge!)
```

Comment:
My bad, should be `inject(Corrector.new(processed_source), :merge!)` otherwise we modify the first corrector, or return `nil` if there are no correctors. Other looks good

Reply:
Fixed!